### DISCUSSION TITLE
Reserved keyword as table name?

### DISCUSSION THREAD

**Discussion by cpryor81** [entechlog/dbt-snow-mask#52] [Q&A]
I ran into an issue where the table name used (which can't be changed) is using a reserved keyword (i.e. "GROUP").  I'm struggling to find a way to identify the table in the source.yml so that it doesn't error. 

I've tried the following: 
"\"GROUP\""
'\"GROUP\"'
"GROUP"
[GROUP]

but none seem to work.  Any thoughts?

**Comment by mjahammel** [comment]
I'm a little late, but does this help any?

https://towardsdatascience.com/debugging-dbt-when-your-table-name-is-a-reserved-keyword-faf81ab6376b
